lib/ua_parser/parsers/device.ex

```diff
@@ -8,18 +8,20 @@ defmodule UAParser.Parsers.Device do
 
   import Base
   @behaviour Base
-
   def parse(nil), do: %Device{}
   def parse({group, match}) do
-    family = Keyword.get(group, :device_replacement)
-
-    family =
+    [family: :device_replacement, brand: :brand_replacement, model: :model_replacement ]
```

Comment:
Could you sort these alphabetically and add spaces after the tuple and list commas?